examples/src/main/java/com/example/ftclibexamples/MecanumDrivingSample.java

```diff
@@ -1,61 +1,57 @@
 package com.example.ftclibexamples;
 
 import com.arcrobotics.ftclib.drivebase.MecanumDrive;
-import com.arcrobotics.ftclib.geometry.Vector2d;
-import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.arcrobotics.ftclib.hardware.RevIMU;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
 import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 
+@TeleOp
 public class MecanumDrivingSample extends LinearOpMode {
 
-    private MecanumDrive driveTrain;
-
-    private MotorEx fL, fR, bL, bR;
-
-    // the motors used here are goBILDA Yellow Jackets, 435 rpm
+    static final boolean FIELD_CENTRIC = false;
```

Comment:
Explain why this variable is there. It might seem self explanatory but best to be as verbose as possible.